### ISSUE TITLE
Connection not established within 15 seconds.

### ISSUE THREAD

**Issue by Bowtell** [JoshIce/GachiBot#4]
The bot has worked for so long, but recently it wasn't used for a few weeks and now trying again to use it I keep getting this error:

![image](https://user-images.githubusercontent.com/6972491/67659608-dbe09680-f9b0-11e9-8736-1077b24f6792.png)

I am a large noob when it comes to this stuff so I have not a clue about what to do
Any help would be fab

**Comment by JoshIce** [comment]
This looks like the bot can't connect to Discord. Try generating a new token for the bot and updating it per the instructions in the readme. Keep me updated.

**Comment by Bowtell** [comment]
Sorry, I probably should have elaborated more in the original post.
The bot can connect and even join the voice channel when a command it typed but it doesn't play the command and just sits in the voice channel.

**Comment by JoshIce** [comment]
Have you tried using several commands? There's currently an issue where the bot cannot play an audio file shorter than ~2 seconds.

**Comment by Bowtell** [comment]
Yeah, have encountered this issue before, this is different. No matter which command is typed it always does the same.

**Comment by JoshIce** [comment]
I just tested my install and I'm not experiencing any issues, so I don't think it's an API change with Discord. I'm not entirely sure what could be causing the issue at the moment, but Discord's servers might be the problem. Could you try adding the bot to another server? Maybe also change the server location to one that's different than your normal server. Mine's working on US West.

**Comment by chaseh80** [comment]
Also having the same problem.
[picture of error](https://i.gyazo.com/67d2536f35ad2f3a0ba5e17c137d57d1.png)
